src/css/application.css

```diff
@@ -212,11 +212,13 @@ body {
   box-shadow: 0 0.1em 0.2em 0 rgba(0, 0, 0, 0.3);
 }
 
+.top-bar__menu-button_export,
```

Comment:
Any reason not to just give the export button the `top-bar__menu-button_primary` class rather than duplicating these properties in the `_export` class?

Generally better imo to keep the actual class definitions/rules simpler and more transparent, and then apply the classes we need in the HTML.

(To be clear, I would still propose keeping the `_export` modifier as well, to apply the rules around margin and border)